### PR TITLE
Import services from ambanum/OpenTermsArchive

### PR DESCRIPTION
**Pull request by Ndpnt**
⚠️ Before merging change the branch from `extract-services` to `master` of the `open-terms-archive` dependency in `package.json`

### CODE REVIEW

**Review by martinratinaud — APPROVED** [review]
Tested on local and it works great. Congrats

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "test": "ota-validate",
+    "test:schema": "npm run test -- --schema-only"
+  },
+  "dependencies": {
+    "ajv": "^6.12.6",
+    "chai": "^4.3.4",
+    "datauri": "^4.1.0",
+    "json-source-map": "^0.6.1",
+    "mime": "^3.0.0",
+    "mocha": "^9.1.3",
```
Comment (MattiSG) [inline package.json:14]: These should be dependencies of OTA, not of the services repository.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: Validate services
```
Comment (MattiSG) [inline .github/workflows/test.yml:1]: Only changed service should be validated, as they were on OTA 🙂

File: README.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
-# OpenTermsArchive services
+# Open Terms Archive - All services
 
-This repo is a temporary repository that will store at first
-- documents that needs fixing (created by [OpenTermsArchive](https://github.com/ambanum/OpenTermsArchive))
-- documents that need to be added (created by [OpenTermsArchive.org](https://github.com/ambanum/OpenTermsArchive.org))
+This repository is to be considered as a database of services declarations used by [Open Terms Archive](https://github.com/ambanum/OpenTermsArchive).
```
Comment (MattiSG) [inline README.md:3]:
```suggestion
The database of services tracked on [Open Terms Archive](https://opentermsarchive.org).
```

File: README.md [hunk 4]
```diff
@@ -1,9 +1,13 @@
...
+You can learn more about [Open Terms Archive](https://github.com/ambanum/OpenTermsArchive) on the [dedicated repository](https://github.com/ambanum/OpenTermsArchive).
```
Comment (MattiSG) [inline README.md:4]:
```suggestion
These service declarations are used by the [Open Terms Archive engine](https://github.com/ambanum/OpenTermsArchive).
```

**Review by MattiSG — CHANGES_REQUESTED** [review]

**Review by MattiSG — APPROVED** [review]
👍

**Comment by MattiSG** [comment]
Reminder to update the main branch in the settings once this PR is merged 🙂

**Pull request merged**